src/app/public/modules/page-summary/page-summary.component.ts

```diff
@@ -11,6 +11,9 @@ import { Subscription } from 'rxjs/Subscription';
   providers: [SkyPageSummaryAdapterService]
 })
 export class SkyPageSummaryComponent implements OnDestroy, AfterViewInit {
+  @ViewChild('keyInfoContainerEl')
```

Comment:
```suggestion
  @ViewChild('keyInfoContainerEl', { read: ElementRef })
```

IE 11 needs the `read` property.

Reply:
done